NAMESPACE

```diff
@@ -279,6 +285,7 @@ export(getGoogleBigQueryDataSets)
 export(getGoogleBigQueryProjects)
 export(getGoogleBigQueryTable)
 export(getGoogleBigQueryTables)
+export(getGoogleTokenForCloudStorage)
```

Comment:
Move it to the right alphabetical location.

Reply:
fixed the order.